docs/guides/migration_guide_facilities_to_metros_devices.md

```diff
@@ -48,9 +48,9 @@ resource "metal_device" "node" {
 }
 ```
 
-To test that the change didn't taint the state, and that the device will not be re-created, you can check if `terraform plan` reports no differences. The terraform state should be up to date as long as the facility in which the device was deployed was contained within the metro.
+To test that the change didn't taint the state, and that the device will not be re-created, you can check if `terraform plan` reports any differences. The terraform state should be up to date as long as the facility in which the device was deployed was contained within the metro.
 
-If the plan diff is not empty, you might have set a metro not containing the facility to which the device was deployed. This might happen if you've used more facilities in the `facilities` list, or you have used "any" facility.
+If the plan diff is not empty, you might have used a metro not containing the facility to which the device was deployed. This might happen if you've used more facilities in the `facilities` list, or you have used "any" facility.
```

Comment:
```suggestion
If the plan diff is not empty, you might have used a metro not containing the facility to which the device was deployed. This might happen if you've used more facilities in the `facilities` list, or you have used the special "any" facility.
```